cognite/client/_api/synthetic_time_series.py

```diff
@@ -5,81 +5,102 @@
+            expressions (Union[str, "sympy.Expr", List[Union[str, "sympy.Expr"]]]): Functions to be calculated. Supports both strings and sympy expressions. Strings can have either the API `ts{}` syntax, or contain variable names to be replaced using the `variables` parameter.
             start (Union[int, str, datetime]): Inclusive start.
-            end (Union[int, str, datetime]): Exclusive end.
-            limit (int): Number of datapoints to retrieve.
+            end (Union[int, str, datetime]): Exclusive end
+            limit (int): Number of datapoints per expression to retrieve.
             variables (Dict[str,Union[str,TimeSeries]]): An optional map of symbol replacements.
             aggregate (str): use this aggregate when replacing entries from `variables`, does not affect time series given in the `ts{}` syntax.
             granularity (str): use this granularity with the aggregate.
 
         Returns:
-            Datapoints: A Datapoints object containing the calculated data.
+            Union[Datapoints, DatapointsList]: A DatapointsList object containing the calculated data.
 
         Examples:
 
             Request a synthetic time series query with direct syntax
 
-                >>> from cognite.client.experimental import CogniteClient
+                >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
-                >>> dps = c.datapoints.synthetic.retrieve(expression="TS{id:123} + TS{externalId:'abc'}", start="2w-ago", end="now")
+                >>> dps = c.datapoints.synthetic.query(expressions="TS{id:123} + TS{externalId:'abc'}", start="2w-ago", end="now")
 
             Use variables to re-use an expression:
 
-                >>> from cognite.client.experimental import CogniteClient
+                >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
                 >>> vars = {"A": "my_ts_external_id", "B": client.time_series.retrieve(id=1)}
-                >>> dps = c.datapoints.synthetic.retrieve(expression="A+B", start="2w-ago", end="now", variables=vars)
+                >>> dps = c.datapoints.synthetic.query(expressions="A+B", start="2w-ago", end="now", variables=vars)
 
             Use sympy to build complex expressions:
 
-                >>> from cognite.client.experimental import CogniteClient
+                >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
                 >>> from sympy import symbols, cos, pi
                 >>> a = sympy.symbols('a')
-                >>> dps = c.datapoints.synthetic.retrieve(pi * cos(a), start="2w-ago", end="now", variables={"a": "my_ts_external_id"},aggregate='interpolation',granularity='1m')
+                >>> dps = c.datapoints.synthetic.query([sin(a), cos(a)], start="2w-ago", end="now", variables={"a": "my_ts_external_id"},aggregate='interpolation',granularity='1m')
```

Comment:
```suggestion
                >>> dps = c.datapoints.synthetic.query(["sin(a), cos(a)"], start="2w-ago", end="now", variables={"a": "my_ts_external_id"},aggregate='interpolation',granularity='1m')
```
doesn't this need quotes?

Reply:
no, I think, cause `sin(a)` and `cos(a)` is two different expressions with `sympy.Expr` type

Reply:
["sin(a)", "cos(a)"]

Reply:
@sanderland , `[sin(a), cos(a)]` is also valid, isn't it?

Reply:
yes since this is sympy